WordPress/src/main/java/org/wordpress/android/ui/posts/mediauploadcompletionprocessors/AudioBlockProcessor.kt

```diff
@@ -0,0 +1,39 @@
+package org.wordpress.android.ui.posts.mediauploadcompletionprocessors
+
+import com.google.gson.JsonObject
+import org.jsoup.nodes.Document
+import org.wordpress.android.util.helpers.MediaFile
+
+class AudioBlockProcessor(localId: String?, mediaFile: MediaFile?) : BlockProcessor(localId, mediaFile) {
+    override fun processBlockContentDocument(document: Document?): Boolean {
+        val audioElements = document?.select(AUDIO_TAG)
+
+        audioElements?.let { elements ->
+            for (element in elements) {
+                // replaces the src attribute's local url with the remote counterpart.
+                element.attr(SRC_ATTRIBUTE, mRemoteUrl)
+            }
+            return true
+        }
+        return false
+    }
+
+    override fun processBlockJsonAttributes(jsonAttributes: JsonObject?): Boolean {
+        val id = jsonAttributes?.get(FileBlockProcessor.ID_ATTRIBUTE)
```

Comment:
NP: We can use the local `ID_ATTRIBUTE` instead of the one from `FileBlockProcessor`

```suggestion
        val id = jsonAttributes?.get(ID_ATTRIBUTE)
```

Reply:
whoops! Thanks for spotting that @mkevins 🙇